coach/src/user/menu.cjsx

```diff
@@ -38,7 +38,7 @@ UserMenu = React.createClass
     @context.navigator.emit('show.student_id', view: 'student_id')
 
   update: ->
-    @forceUpdate() if @isMounted()
+    @forceUpdate()
```

Comment:
yes, I don't see anywhere that calls it either.  Am tempted to remove it but maybe I've missed something.  Right now it's just a warning to call `forceUpdate` on an unmounted component, but would be an error to remove a method and then it's called somewhere